domains/rameez.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Personal website of rameez",
+  "repo": "https://github.com/rameezrami",
+  "owner": {
+    "username": "rameezrami",
+    "email": "rameespu@gmail.com@gmail.com"
+  },
+  "record": {
+    "URL": "http://rameez.is-a.dev"
```

Comment:
You need a cname/a record (or a url redirection to an existing page) for this to be a valid record

I'd suggest you create a repo with the name `rameezrami.github.io`. You can take a look at the [github pages guide](https://guides.github.com/features/pages/).